.github/workflows/e2e.yml

```
@@ -6,27 +6,33 @@ jobs:
   e2e:
     runs-on: ubuntu-latest
     name: End-to-end tests
+    concurrency:
+      group: react-e2e
+      cancel-in-progress: false
+    env:
+      PLAYWRIGHT_BROWSERS_PATH: ./pw-browsers
     steps:
       - uses: actions/checkout@v3
 
       - name: 💾 Cache Dependencies
         uses: actions/cache@v3
         with:
-          path: ./node_modules
+          path: |
+            ./node_modules
+            ./pw-browsers
           key: ${{ runner.os }}-${{ matrix.node }}-modules-${{ hashFiles('**/yarn.lock') }}
 
       - name: 🔨 Install Dependencies
-        run: yarn install --frozen-lockfile --ignore-engines --ignore-scripts
-
-      - name: ⚗️ End-to-end tests
         run: |
+          yarn install --frozen-lockfile --ignore-engines --ignore-scripts
           npx playwright install
```

```suggestion
          npx playwright install chromium
```
Let's skip the installation of the other two as those are not needed (for the time being).

---

Other thing (consider it a nit) - would it be faster to move this command to its own step and running it only if the cache-hit is `false`?

---

Playwright actually checks if binaries are already up-to-date and only downloads them as necessary. So adding the path with browser binaries to cache is enough.